site/student-lectures/cultural-algorithms/cultural-algorithms.rst

```diff
@@ -0,0 +1,255 @@
+*******************
+Cultural Algorithms
+*******************
+
+Introduction
+============
+* It is a branch of evolutionary computation
+* Unlike genetic algorithms, cultural algorithms have not only **population space** but also **belief space**
+* Here shows the components of cultural algorithms and its structure [#]_:
+
+.. figure:: ca_components.png
+    :align: center
+
+
+* Object() is the fitness function
+* The most important part of the cultural algorithm is:
+
+    * Accept() picks top n individuals from population
+    * Update() updates belief space based on the individuals pick from Accept()
+    * Influence() is to use the knowledge in belief space to update population
```

Comment:
If you want to make it look "code like. use double back ticks ` to make it look more like code.